### ISSUE TITLE
Office.context.mailbox.item.body.setAsync not work for Mac Outlook 16.44(20110306)

### ISSUE THREAD

**Issue by hello-chenchen** [OfficeDev/office-js#1487]
demo code:
```
var setText = function () {
    return new Office.Promise(function (resolve) {
        Office.context.mailbox.item.body.setAsync(
            'test text',
            {coercionType: Office.CoercionType.Text}, (function (result) {
            console.log(result);
            }));
        });
};
```

get error message: 
```
error: Object

code: 5001

message: "An internal error has occurred."

name: "Internal Error"

Object Prototype

status: "failed"
```

**Comment by exextoc** [comment]
Thanks for reporting this issue. We are already aware of this issue and working on fixing this.

**Comment by exextoc** [comment]
We have fixed the issue with Body API and fix is available in build >= 16.44(20111502). Depending on the release channel a customer is on, getting an updated build can take anywhere from 4-8 weeks.
